plugins/karbon.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: karbon
+spec:
+  version: v0.2.0
+  homepage: https://github.com/nutanix/kubectl-karbon
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_darwin_amd64.tar.gz
+    sha256: 866b4398547173433163e33912c2fcae91415b6fef2c434a498bceb2afa69fcf
+    bin: kubectl-karbon
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: arm64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_darwin_arm64.tar.gz
+    sha256: d75a9429a8b35e48d2dc4d110c9d0f5dfbc07466b51f993d61fe2c3f664538ea
+    bin: kubectl-karbon
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_linux_amd64.tar.gz
+    sha256: f7c27a7f00ca582c63051c7ce999b581732508ca566612aa4cd47ea5509d6cf3
+    bin: kubectl-karbon
+  - selector:
+      matchLabels:
+        os: linux
+        arch: arm64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_linux_arm64.tar.gz
+    sha256: 64e522186956b17ab7ae6e05217b3c7055a3feb52c18f1d394110d0608033242
+    bin: kubectl-karbon
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/nutanix/kubectl-karbon/releases/download/v0.2.0/kubectl-karbon_v0.2.0_windows_amd64.tar.gz
+    sha256: 8d8485d868e8f604e43c5926523ef77cccab11688daff1c5ac079a3079c8e8e4
+    bin: kubectl-karbon
```

Comment:
I believe this needs to be an .exe?

You might wanna try KREW_OS=windows and see how it gets symlinked (it needs an exe suffix i believe)